projects/envoy/find_corpus.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python
+import os, sys, re
+fuzzer_target = sys.argv[1].split('/')
+directory, fuzz_test = fuzzer_target[:-1], fuzzer_target[-1]
+directory_string = "/".join(directory)
+path = "../envoy/" + directory_string + "/BUILD"
```

Comment:
This is a bit evil (what if the caller moves to a different relative path before invoking?), but it's fine for the duct tape we're doing in the oss-fuzz Docker image.